examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -221,28 +221,22 @@ extension BlobDownloadViewController: UITableViewDelegate, UITableViewDataSource
 }
 
 extension BlobDownloadViewController: TransferDelegate {
-    func client(forRestorationId restorationId: String) -> PipelineClient? {
-        guard restorationId == "download" else { return nil }
+    func client(forRestorationId _: String) -> PipelineClient? {
         return blobClient
     }
 
-    func options(forRestorationId restorationId: String) -> AzureOptions? {
```

Comment:
Got rid of this. I serialize the options to a JSON string and work with that. I can't simply make them Transformable because they would need to work with `@objc`. It's a lot lower bar to make something `Codable` vice `@objc`, so I just handle this in the SDK. We can revisit later.

Reply:
I think this is a good approach!